js/apps/system/_admin/aardvark/APP/react/src/views/views/forms/inputs/CopyFromInput.tsx

```diff
@@ -32,38 +32,48 @@ const CopyFromInput = ({ views, dispatch, formState }: CopyFromInputProps) => {
   const initalViewOptions = filterAndSortViews(views);
   const [viewOptions, setViewOptions] = useState(initalViewOptions);
   const [selectedView, setSelectedView] = useState(viewOptions[0]);
-  const { encoded: encodedSelectedView } = encodeHelper(selectedView.value);
+  const { encoded: encodedSelectedViewName } = encodeHelper(selectedView.value);
   const { data } = useSWRImmutable(
-    `/view/${encodedSelectedView}/properties`,
+    `/view/${encodedSelectedViewName}/properties`,
     path => getApiRouteForCurrentDB().get(path)
   );
-  const fullView = data ? data.body : selectedView;
+  let viewToCopy = data ? data.body : selectedView;
   const { setCurrentField } = useLinksContext();
 
   useEffect(() => {
     setViewOptions(filterAndSortViews(views));
   }, [views]);
 
   const copyFormState = () => {
-    validateAndFix(fullView);
-    Object.assign(
-      fullView,
-      pick(
-        formState,
-        "id",
-        "name",
-        "primarySort",
-        "primarySortCompression",
-        "storedValues",
-        "writebufferIdle",
-        "writebufferActive",
-        "writebufferSizeMax"
-      )
+    validateAndFix(viewToCopy);
+    let nullifiedLinks = {} as any;
```

Comment:
```suggestion
    const nullifiedLinks = {} as any;
```

I don't see any reassignments. I think we can use `const` ?